### PR TITLE
Geojson to csv conversion test 

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**
For testing the function added in #1485 . A [script](https://github.com/weecology/retriever-recipes/pull/74) is added in retriever-recipies for using this testing function

### CODE REVIEW

**Comment by ashishpriyadarshiCIC** [comment]
To test this use the command `retriever install csv lake-county-illinois-cancer-rates` and here is the output

image

File: retriever/lib/engine_tools.py [hunk 1]
```diff
@@ -17,6 +17,10 @@
 import xml.etree.ElementTree as ET
 import os
 import csv
+try:
+    import geopandas
+except:
```
Comment (apoorvaeternity) [inline retriever/lib/engine_tools.py:22]: `except ModuleNotFoundError`

**Review by apoorvaeternity — CHANGES_REQUESTED** [review]
Couple of comments but looks good.

File: test/test_retriever.py [hunk 2]
```diff
@@ -12,6 +14,12 @@
 from retriever.lib.engine_tools import getmd5
 from retriever.lib.engine_tools import xml2csv
 from retriever.lib.engine_tools import json2csv
+
+try:
```
[the remark this answers is not in the document]
Comment (ashishpriyadarshiCIC) [reply test/test_retriever.py:18]: The tests on Travis and Appveyor are showing issues in installing geopandas library.

File: test/test_regression.py [hunk 3]
```diff
@@ -274,31 +274,31 @@ def test_download_regression(dataset, expected, encodings):
...
-    # table names of the installed data packages in
-    # script_table_registry
-    
-    workdir = tmpdir.mkdtemp()
-    workdir.chdir()
-    rt.install_csv("iris")
-    wine_data = rt.fetch("wine-composition")
-    assert "iris" not in wine_data.keys()
-    os.chdir(retriever_root_dir)
+# def test_fetch():
```
[the remark this answers is not in the document]
Comment (ashishpriyadarshiCIC) [reply test/test_regression.py:277]: The datasets were not online when this PR was made initially, It's online now so I will uncomment it

**Comment by ashishpriyadarshiCIC** [comment]
@apoorvaeternity @henrykironde Ready for review

File: .travis.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ services:
 
 # command to run tests using py.test
 script:
-  - docker-compose run --service-ports python_retriever pytest -v --cov=retriever
+  - docker-compose run --service-ports python_retriever pytest -v -k "not test_geojson2csv" --cov=retriever
```
Comment (henrykironde) [inline .travis.yml:9]: Adding a comment here would be nice and also in the try blocks 

```
try:
     # Geopanda installation is not smooth on the CI tests platforms
     import geopandas
 except ModuleNotFoundError:
     pass
```

**Review by apoorvaeternity — APPROVED** [review]
LGTM!!!

**Comment by henrykironde** [comment]
Check out the comments and update each of them. Looks good but need to take care of the commetns

**Pull request merged**